lib/package_contents.rb

```diff
@@ -0,0 +1,13 @@
+module PackageContents
+  def components
+    File.readlines("node_modules/govuk-frontend/dist/govuk/components/_index.scss")
+      .map { |line| /@import "(?<component>[a-z-]*)\/index";/.match(line)&.[](:component) }
```

Comment:
Thinking we should probably throw if we end up with an empty list? If we changed the structure or format of the index file (or moved to the Sass module system!) we'd want this to error rather than end up with an example with no components or overrides? 🤔